README.rst

```diff
@@ -24,7 +24,7 @@ Why use ScanCode?
   and more both in **source code** and **binary** files.
 
 - ScanCode provides the **most accurate license detection engine** and does a
-  full comparison (aka. diff or red line) between a database of license texts
+  full comparison (also known as, diff or red line comparison) between a database of license texts
```

Comment:
Thank you for the update... can you also remove the comma in  `also known as,`?

Reply:
Okay